src/_breakpoints.scss

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
These mixins provide wrap the media queries for us.  A couple of things worth noting:

* the grid definition remains the same (12 fluid columns and gutters) - we do have the ability to change grid definition, but currently there is no requirement to do that (in speaking with Alex)

* there is no mixin for the smallest size (0-767px) since "mobile first" implies it's the default.  Consumers write their SCSS/CSS for this size as they normally would (without any media query).

* there are no max-widths defined for media queries.  This is to enable consumers to only implement different layout for the breakpoints as required.  For example, is small is not required, the user would see the mobile view up to 991px, and then they would see the medium/desktop view.  A consequence to that is that consumers must be careful when setting up their CSS because some settings could potentially get inherited.